### PR TITLE
[New3D] Promote New3D to beta

### PR DESCRIPTION
**Pull request by jhurliman**
**User-Facing Changes**

- A new panel type, `3D (Beta)`, is available for testing

**Description**

Remove the experimental name and flag from New3D

### CODE REVIEW

File: packages/studio-base/src/panels/index.ts [hunk 1]
```diff
@@ -56,6 +56,15 @@ const builtin: PanelInfo[] = [
     thumbnail: threeDimensionalVizThumbnail,
     module: async () => await import("./ThreeDimensionalViz"),
   },
+  {
+    title: "3D (Beta)",
+    type: "3D",
+    description:
+      "Display visualization markers and models in a 3D scene using a new rendering engine.",
```
Comment (amacneil) [inline packages/studio-base/src/panels/index.ts:63]: At first glance this text is very similar to the old 3d panel (except the last few words). While it is in beta, it's probably best to get straight to the point, like:

```suggestion
      "Test our next-generation 3D panel, using a new rendering engine.",
```

**Comment by amacneil** [comment]
👍 lgtm. I think we should disable `render stats` by default

**Review by defunctzombie — APPROVED** [review]
Agree with @amacneil comment to drop the render stats hover widget. You could make it a sidebar setting :D

**Comment by amacneil** [comment]
It already is a sidebar setting :)

**Comment by defunctzombie** [comment]
> It already is a sidebar setting :)

😮

**Pull request merged**